docs/src/pages/core-concepts/collections.md

```diff
@@ -3,204 +3,193 @@ layout: ~/layouts/Main.astro
 title: Collections
 ---
 
-**Collections** are a special type of [Page](/core-concepts/astro-pages) that help you generate multiple pages from a larger set of data. Example use-cases include:
+**Collections** are a special type of [Page](/core-concepts/astro-pages) that can generate multiple pages from a larger set of data. Example use-cases include:
 
-- Pagination: `/posts/1`, `/posts/2`, etc.
-- Grouping content by author: `/author/fred`, `/author/matthew`, etc.
-- Grouping content by some tag: `/tags/red`, `/tags/blue`, etc.
-- Working with remote data
-- Mixing remote and local data
+- Automatic pagination: `/posts/1`, `/posts/2`, etc.
+- Grouping content: `/author/fred`, `/author/matthew`, etc.
+- Generating one page per item: `/pokemon/pikachu`, `/pokemon/charmander`, etc.
+- Generating pages from remote data
+- Generating pages from local data
 
 **Use a Collection when you need to generate multiple pages from a single template.** If you just want to generate a single page (ex: a long list of every post on your site) then you can just fetch that data on a normal Astro page without using the Collection API.
 
 ## Using Collections
 
-To create a new Astro Collection, you must do three things:
+To create a new Astro Collection, you must do two things:
 
 1. Create a new file in the `src/pages` directory that starts with the `$` symbol. This is required to enable the Collections API.
 
+- Example: `src/pages/$tags.astro` -> `/tags/:tag`
 - Example: `src/pages/$posts.astro` -> `/posts/1`, `/posts/2`, etc.
-- Example: `src/pages/$tags.astro` -> `/tags/:tag` (or `/tags/:tag/1`)
 
-2. Define and export the `collection` prop: `collection.data` is how you'll access the data for every page in the collection. Astro populates this prop for you automatically. It MUST be named `collection` and it must be exported.
-
-- Example: `const { collection } = Astro.props;`
-
-3. Define and export `createCollection` function: this tells Astro how to load and structure your collection data. Check out the examples below for documentation on how it should be implemented. It MUST be named `createCollection` and it must be exported.
+2. Define and export a `createCollection` function: this tells Astro what pages to generate. It **MUST** be named `createCollection` and it must be exported. Check out the examples below for documentation on how it should be implemented.
```

Comment:
Perhaps mention this createCollection should be inserted in the $ page created above?
Otherwise people might confuse this needs to be added in a config file?